modules/custom/openy_search/openy_search_api/openy_search_api.install

```diff
@@ -72,3 +72,35 @@ function openy_search_api_uninstall() {
     }
   }
 }
+
+/**
+ * Enable Search API Solr legacy if needed.
+ */
+function openy_search_api_update_8001() {
```

Comment:
It might be a good idea to move this to `openy.install` as it covers the needs of many modules, not only the Open Y Search API.

Reply:
makes sense

Reply:
but probably just needs duplicated hook_update_N in modules that are using Search API ( Activity Finder as of now only )
Anything else I've missed @AndreyMaximov

Reply:
Added the same thing to Activity Finder, @podarok 
https://github.com/ymcatwincities/openy/pull/2177/commits/62d88a82f631f2c65943f2cc7af42c0a0601e43e